violation/models.py

```diff
@@ -171,6 +171,7 @@ def related_entities(self):
 
 
 @versioned
+@sourced
```

Comment:
These attributes are, in fact, all sourced. (Violation records come with a single source that the importer associates with each data point.) The complex field module expects sourced data points to be declared sourced and [will not return any sources – even if they exist – if data points are not marked as such](https://github.com/security-force-monitor/complex_fields/blob/328aef79a458fb568f9917929f9817c8c73f729c/complex_fields/models.py#L203-L207). 

This change makes sourcing more explicit and allows me to use `get_sources` in the new test confirming all data points have evidence. Unfortunately, the source creation form behaves differently in that it only expects and creates a source for description, so this change breaks those tests (and the UI).

Per @tlongers, this UI is no longer in use. I have marked the failing tests for skipping, and will open a separate issue to clean up the unused code.